runtime/src/fips.rs

```diff
@@ -5,55 +5,39 @@ use caliptra_drivers::CaliptraError;
 use caliptra_drivers::CaliptraResult;
 use caliptra_kat::{Ecc384Kat, Hmac384Kat, Sha256Kat, Sha384AccKat, Sha384Kat};
 use caliptra_registers::mbox::enums::MboxStatusE;
-use zerocopy::{AsBytes, FromBytes};
 
-use crate::Drivers;
+use crate::{Drivers, MailboxResp, MailboxRespHeader, FipsVersionResp};
 
-pub struct FipsModule;
-
-#[repr(C)]
-#[derive(Clone, Debug, Default, AsBytes, FromBytes)]
-pub struct VersionResponse {
-    pub mode: u32,
-    pub fips_rev: [u32; 3],
-    pub name: [u8; 12],
-}
-
-impl VersionResponse {
-    pub const NAME: [u8; 12] = *b"Caliptra RTM";
-    pub const MODE: u32 = 0x46495053;
-
-    pub fn new(_env: &Drivers) -> Self {
+impl FipsVersionResp {
```

Comment:
Consider placing struct and impl in the same file.

Reply:
I can change this. I was on the fence about which approach made the most sense here. This was an attempt to separate the message formats and implementation. But I can see how this may be less intuitive.

Reply:
I don't feel super strongly about this. But I think in the future a nice pattern here could be

```rust
// <mailbox_cmd_impl>.rs
struct Cmd {
   ...
}

impl Cmd {
    fn execute(&self) -> CaliptraResult<Resp> {}
}
```

Then you can call it like this

```rust
let cmd = Cmd{};
cmd.execute();
```

So that would probably mean putting each command struct in the corresponding implementation file. But again, not a big deal either way.

Reply:
+1 split

Reply:
I just pushed a patch that I think is what you were talking about here. I followed the existing example in the info implementation.

Reply:
Let's leave it how it is now. I have some other ideas for refactoring, but this PR is already very big. I think it would be good to get these changes in as-is.